docs/api/download-item.md

```diff
@@ -1,5 +1,7 @@
 # DownloadItem
 
+> Trigger file downloads from remote sources.
```

Comment:
The download is triggered by `webContents.downloadURL`, `DownloadItem` is just a class representing the download item.